tutorials/beginner/installing.md

```diff
@@ -1,27 +1,55 @@
 ---
 layout: tutorial-beginner
+title: Installing Extension
 ---
 
 {% include base.html %}
 
-# To be done
+# Installing an Extension
 
-This part is not yet written.
+One of the strengths of openHAB is that new capabilities can easily be added by installing extensions. These capabilities can be a technology (like Z-Wave, Philips HUE, Sonos, ...), user interfaces, database connections and others.
 
-.
+Especially at the beginning of an openHAB installation one of the most common activities is to include existing home automation technologies. Therefore we use a **binding** to learn how to add extensions.
 
-.
+The workflow for adding a binding normally consists of three steps:
 
-.
+1. [Add new binding](#add-new-binding)
+2. [Scan for new things](#scan-for-new-things)
+3. [Configure things](#configure-things)
 
-.
 
-.
+## Prerequisites
 
-.
+* Access to the [PaperUI](uis.html/#the-paper-ui)
 
-.
 
-.
+## Add New Binding
 
-Sorry...
\ No newline at end of file
+| 1. |  ```http://192.168.20.3:8080/start/index```   | Point your browser to the PaperUI. Please adjust the IP adress to that of your openHab server. |
```

Comment:
Please be careful with the spelling. It was decided for "openHAB" to be the only valid version.